### PR TITLE
ci(tests): add step to uninstall packages

### PR DESCRIPTION
**Pull request by wizard-28**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -88,6 +88,25 @@ jobs:
...
+          export DEBIAN_FRONTEND=noninteractive
+          export GITHUB_ACTIONS=true
+          for changed_file in ${{ steps.files.outputs.added_modified }}; do
+            if [[ ${changed_file} == *".pacscript" ]]; then
+              pacscript_file=`basename "${changed_file}"`
+              package_name="${pacscript_file/.pacscript/ }"
+              package_dir=`dirname "${changed_file}"`
+              echo "Running pacstall -R for ${package_name}..."
+              cd ${package_dir}
+              pacstall --disable-prompts -R ${pacscript_file}
```
Comment (Henryws) [inline .github/workflows/tests.yml:104]: Is this the pacscript variable without `.pacscript`?

Comment (wizard-28) [reply .github/workflows/tests.yml:104]: No, I think `package_name` would be the correct variable. You can suggest the correct code.

Comment (Henryws) [inline .github/workflows/tests.yml:104]:
```suggestion
              pacstall --disable-prompts -R ${package_name}
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -166,3 +185,22 @@ jobs:
...
```
Comment (Henryws) [inline .github/workflows/tests.yml:201]:
```suggestion
              pacstall --disable-prompts -R ${package_name}
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by Henryws — APPROVED** [review]

**Pull request merged**